src/ks-app/package.json

```diff
@@ -21,7 +21,7 @@
     "@angular/platform-browser": "^5.0.0",
     "@angular/platform-browser-dynamic": "^5.0.0",
     "@angular/router": "^5.0.0",
-    "@clr/angular": "file:../../dist/clr-angular/clr-angular-0.11.7-patch.1.tgz",
+    "@clr/angular": "file:///Users/mhippely/Projects/clarity/fork/dist/clr-angular/clr-angular-0.11.7-patch.1.tgz",
```

Comment:
I just saw this and will push a fix for it.

Reply:
Ok, I see another problem here. The file is always pointing to a specific version number, which might be changed but is sometimes not updated when we do a release. Package-lock also points to this.

Can we point to a directory instead? We build a tgz asset, but don't we also have a directory with the source files we could point to instead?

Reply:
I thought I fixed this last night. 
The ks-app has issues linking to source files directly. This is why the tgz has been used since we switched to webpack.